### DISCUSSION TITLE
Двустороннее связывание | Angular с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/angdev.ru#12] [Announcements]
# Двустороннее связывание | Angular с примерами кода

Механизм двустороннего связывания (angular two way binding) используется, когда необходимо отобразить свойство в шаблоне (одностороннее связывание) и обновить его при изменении значения (возникновение события) без перезагрузки страницы

https://angdev.ru/doc/two-way-data-binding/

**Comment by Parchuk** [comment]
Good!
